manager/integration/tests/test_node.py

```diff
@@ -1003,21 +1005,22 @@ def test_replica_cleanup(client):  # NOQA
     for replica in volume["replicas"]:
         data_paths.append(replica["dataPath"])
 
+    dockerd_pid = find_dockerd_pid() or "1"
     # data path should exist now
     for data_path in data_paths:
-        assert subprocess.call(["nsenter",
-                                "--mount=/host/proc/1/ns/mnt",
-                                "--net=/host/proc/1/ns/net",
-                                "bash", "-c", "ls " + data_path]) == 0
+        assert subprocess.call(
```

Comment:
Can we use `exec_nsenter` instead?

Reply:
No. Using exec_nsenter() will cause error.

Reply:
why it would cause an error?